spec/features/admin/shipping_methods_spec.rb

```diff
@@ -103,9 +103,8 @@
 
       click_button I18n.t("actions.create")
 
-      expect(page).to have_no_button I18n.t("actions.create")
-      message = "Shipping method \"Teleport\" has been successfully created!"
-      expect(page).to have_flash_message message
+      expect(page).to have_content "Editing Shipping Method"
+      expect(flash_message).to eq 'Shipping method "Teleport" has been successfully created!'
```

Comment:
I also changed this expectation to be in line with the syntax used throughout our other feature specs.

Reply:
Let's not forget to use I18n in assertions as agreed on https://github.com/openfoodfoundation/openfoodnetwork/wiki/Code-Conventions#always-use-i18nt-in-spec-assertions.